apps/sample-app/src/components/utils.tsx

```diff
@@ -76,13 +77,16 @@ export const handleMail = async (a: AvatarItem): Promise<void> => {
     type: mail.ComposeMailType.New,
     toRecipients: [`${a.id}`],
   };
-  const result = await mail.composeMail(mailParams);
-  return alert(result);
+  await mail.composeMail(mailParams);
 };
-export const handleCalendar = async (a: AvatarItem): Promise<void> => {
-  const calendarParams: calendar.ComposeMeetingParams = {
-    attendees: [`${a.id}`],
-  };
-  const result = await calendar.composeMeeting(calendarParams);
-  return alert(result);
+
+export const handleOpenMailItem = async (emailItem: MessageListItem): Promise<void> => {
+  if (!mail.isSupported()) {
+    alert('open mail item is not supported');
```

Comment:
I don't know what the design for this sample is but you might consider not showing the UX that would trigger this handler if the mail functionality isn't supported, rather than teasing the button and then letting the user find out that it doesn't do anything only after its been clicked.